media-libs/aubio/files/aubio-0.4.9-gcc-14.patch

```diff
@@ -0,0 +1,36 @@
+From 95ff046c698156f21e2ca0d1d8a02c23ab76969f Mon Sep 17 00:00:00 2001
```

Comment:
Please always include references at the top of patches, per https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions.

Reply:
Hello @thesamesam,

Thank you for the review!
I've updated this patch to include references.